The `not` dependency is for an LLVM tool. We don't really care about tests enough to add a tool_requires on llvm-core for this.

diff --git a/stablehlo/tests/CMakeLists.txt b/stablehlo/tests/CMakeLists.txt
--- a/stablehlo/tests/CMakeLists.txt
+++ b/stablehlo/tests/CMakeLists.txt
@@ -19,14 +19,6 @@
   MAIN_CONFIG
   ${CMAKE_CURRENT_SOURCE_DIR}/lit.cfg.py
 )
-add_lit_testsuite(check-stablehlo-tests "Running the tests/ suite"
-  ${CMAKE_CURRENT_BINARY_DIR}
-  DEPENDS
-  FileCheck not
-  stablehlo-opt
-  stablehlo-translate
-)
-add_dependencies(check-stablehlo-quick check-stablehlo-tests)
 
 set(LLVM_TARGET_DEFINITIONS TestUtils.td)
 mlir_tablegen(TestUtils.h.inc -gen-pass-decls -name HloTest)
